bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -460,6 +553,176 @@ impl FromHexStr for Height {
     }
 }
 
+impl ops::Add<relative::Height> for Height {
+    type Output = Height;
+    #[inline]
+    fn add(self, rhs: relative::Height) -> Self::Output {
+        match self.checked_add(rhs.value().into()) {
+            Some(height) => height,
+            None => panic!("attempted to add height with overflow"),
+        }
+    }
+}
+
+impl ops::Add<u32> for Height {
```

Comment:
This absolutely goes against the idea of having strong newtypes. I really, really, really don't like this.